db/c.cc

```diff
@@ -2765,6 +2765,12 @@ void rocksdb_ratelimiter_destroy(rocksdb_ratelimiter_t *limiter) {
   delete limiter;
 }
 
+void rocksdb_options_set_row_cache(rocksdb_options_t* opt, rocksdb_cache_t* cache) {
+  if(cache) {
```

Comment:
NIt: how about `opt->rep.row_cache = cache ? cache->rep : nullptr`?

Reply:
Thanks for your suggestion, I will update PR

Reply:
Sorry, on second thought, this may not be the desired behavior. What if opt->rep.row_cache is not nullptr before calling this function?

Reply:
Yeah, I was wondering the same thing, shall I leave it as it is?

Reply:
SGTM.

Reply:
Awesome!